lib/style-formatters/scss.js

```diff
@@ -110,15 +111,17 @@ module.exports = (symbols = [], options = {}) => {
         warnings: []
     });
 
+    let contentOut = fs.readFileSync(path.join(__dirname, '../templates/styles.scss'), 'utf8')
+        .replace(/\/\* VAR_SPRITES \*\//g, options.variables.sprites.trim())
+        .replace(/\/\* VAR_SIZES \*\//g, options.variables.sizes.trim())
+        .replace(/\/\* VAR_VARIABLES \*\//g, options.variables.variables.trim())
+        .replace(/\/\* VAR_MIXIN \*\//g, options.variables.mixin.trim())
+        .replace(/\/\* SPRITES \*\//g, output.sprites.map((sprite) => `${indent()}${sprite}`).join(',\n').trim() || '/* EMPTY */')
+        .replace(/\/\* SIZES \*\//g, output.sizes.filter(Boolean).map((size) => `${indent()}${size}`).join(',\n').trim() || '/* EMPTY */')
+        .replace(/\/\* VARIABLES \*\//g, output.variables.filter(Boolean).map((variable) => `${indent()}${variable}`).join(',\n').trim() || '/* EMPTY */');
+
     return {
         warnings: output.warnings,
-        content: fs.readFileSync(path.join(__dirname, '../templates/styles.scss'), 'utf8')
-            .replace(/\/\* VAR_SPRITES \*\//g, options.variables.sprites.trim())
-            .replace(/\/\* VAR_SIZES \*\//g, options.variables.sizes.trim())
-            .replace(/\/\* VAR_VARIABLES \*\//g, options.variables.variables.trim())
-            .replace(/\/\* VAR_MIXIN \*\//g, options.variables.mixin.trim())
-            .replace(/\/\* SPRITES \*\//g, output.sprites.map((sprite) => `${indent()}${sprite}`).join(',\n').trim() || '/* EMPTY */')
-            .replace(/\/\* SIZES \*\//g, output.sizes.filter(Boolean).map((size) => `${indent()}${size}`).join(',\n').trim() || '/* EMPTY */')
-            .replace(/\/\* VARIABLES \*\//g, output.variables.filter(Boolean).map((variable) => `${indent()}${variable}`).join(',\n').trim() || '/* EMPTY */')
+        content: typeof options.callback === 'function' ? options.callback(contentOut) : contentOut
```

Comment:
Same comments as for the CSS formatter. I do like the separate variable definition in this case so let's do that for all of them but instead use `const content` and just use `content` in the return object.